.github/workflows/ui-test.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and Run Docker Image
+
+on:
+    push:
+      branches: [ "main" ]
+      paths:
+        - 'src/**'
+    pull_request_target:
+      branches: [ "main" ]
+      paths:
+        - 'src/**'
+    workflow_dispatch:
+    schedule:
+      - cron: '0 0 1-31/15 * *'
+
+jobs:
+  build-and-run:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Start Docker Compose
+      run: docker-compose -f dev-docker-compose.yml up --build -d
+
+    - name: Check Docker container exit status
+      run: |
+        if [ "$(docker inspect -f '{{.State.Running}}' openlit)" == "false" ]; then
```

Comment:
maybe add a step for some hosted asset or path, like https://github.com/openzipkin/zipkin/blob/master/build-bin/docker-compose-zipkin.yml?